.github/workflows/regressions.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import datetime
+import os
+import subprocess
+
+dir_path = os.path.dirname(os.path.abspath(__file__))
+regressions_dir = os.path.join(dir_path, 'regression_yamls')
+
+from mcli import RunConfig, create_run
+
+
+def get_configs(cluster: str, mpt_7b_ckpt_path: str, wandb_entity: str,
```

Comment:
can `mpt_7b_ckpt_path` live inside the yaml? 

Or is the idea that you might want to pass in separate version of  the mpt_7b_ckpt_path

Reply:
The idea was to not put our cloud storage paths in a public repo. Probably not the most important thing in the world, but would be nice to parametrize it.